hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -233,11 +264,59 @@ cdef class PureMarketMakingStrategy(StrategyBase):
 
     @property
     def order_level_spread(self) -> Decimal:
-        return self._order_level_spread
+        '''
+        This is only kept to prevent scripts from breaking and 
+        start warning about deprecation of this property for script users
+        bid_order_level_spread is used as the return
+        '''
+        current_tick = self._current_timestamp
+        last_tick = self._last_order_level_warning_timestamp
+        
+        if current_tick > last_tick:
+            self._last_order_level_warning_timestamp = current_tick + self._status_report_interval
+            self.log_with_clock(
+                logging.WARNING, 
+                "order_level_spread is deprecated. "
+                "Please use bid_order_level_spread and ask_order_level_spread. "
+                "Returning bid_order_level_spread")
+        return self._bid_order_level_spread[0]
 
     @order_level_spread.setter
-    def order_level_spread(self, value: Decimal):
-        self._order_level_spread = value
+    def order_level_spread(self, value: Decimal) -> None:
+        '''
+        Kept to prevent user script breaking
+        '''
+        current_tick = self._current_timestamp
+        last_tick = self._last_order_level_warning_timestamp
+        
+        if current_tick > last_tick:
+            self._last_order_level_warning_timestamp = current_tick + self._status_report_interval
+            self.log_with_clock(
+                logging.WARNING, 
+                "order_level_spread is deprecated. "
+                "Please use bid_order_level_spread and ask_order_level_spread. "
+                "Setting both ask_order_level_spread and bid_order_level_spread as [value]")
+
+            self.bid_order_level_spread = [value]
+            self.ask_order_level_spread = [value]
+
+    @property
+    def bid_order_level_spread(self) -> List[Decimal]:
```

Comment:
The property name should be `bid_order_level_spreads`.

Reply:
updated all order_level_spread to order_level_spreads